Assets/RedBlueGames/MulliganRenamer/Editor/Locale/Locale.cs

```diff
@@ -0,0 +1,53 @@
+﻿/* MIT License
+
+Copyright (c) 2019 Murillo Pugliesi Lopes, https://github.com/Mukarillo
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+
+namespace RedBlueGames.MulliganRenamer
+{
+    using UnityEngine;
+
+    [System.Serializable]
+    public class Locale
```

Comment:
Minor thing but my particular conventions do suggest adding Summary comments to class declarations.

Reply:
Ok, will fix in the next commit.
Do you want me to do also the Summaries for the public methods?

Reply:
I'm really torn on the value of them, to be honest. But it would be consistent with the style to add them. If you don't, I don't mind filling them in later.